framework/src/modules/http_api/controllers/transactions.js

```diff
@@ -15,12 +15,13 @@
 'use strict';
 
 const _ = require('lodash');
-const swaggerHelper = require('../../helpers/swagger');
-const ApiError = require('../../helpers/api_error');
+const swaggerHelper = require('../helpers/swagger');
+// TODO Reference http_module for api_error.js and remove it from chain module
+const ApiError = require('../../../../src/modules/chain/helpers/api_error');
```

Comment:
should this use the api module since it's used?

Reply:
if using `ApiError` from HTTP API module, checking `instanceOf` against same in Chain fails. I will be moving helpers files to one module or another so this kind of things don't happen and also to remove duplicate files. Actually, this should be as you said but only after `ApiError` and `ApiCodes` are extracted from Chain module.